src/resource/note/note.controller.ts

```diff
@@ -27,20 +27,20 @@ export class NoteController {
     constructor(private notesService: NoteService) {}
 
     @Get()
-    @FetchUserNotesApi()
-    fetchUserNotes(@GetUser() user: UserEntity) {
-        return this.notesService.fetchUserNotes(user);
+    @GetUserNotesApi()
+    getUserNotes(@GetUser() user: UserEntity): Promise<NoteDTO[]> {
+        return this.notesService.fetchByUser(user);
```

Comment:
I forgot to mention that if TypeScript infer type properly if you hover your mouse on function then you don't need to declare type explicitly